### ISSUE TITLE
MS Teams Invite: "ü" completes selection instead of being a character

### ISSUE THREAD

**Issue by Datrogh** [MicrosoftDocs/msteams-docs#2968]
While creating an invite for a meeting in MS Teams, I try to invite a colleague whose name is "Schüssler". When I enter the "ü", instead of selecting the "Schüssler" from the list, it auto completes the currently selected colleague (in this case "Schimmelschmidt")
This results in an inability to invite said colleague without resorting to tricks, like copy/paste the name from somewhere else.

**Comment by Nikitha-MSFT** [comment]
when you type a letter in the invite section it will provide the the list with the filter with the provided letter. You need to select the user from the filtered list or you need to enter the mail id. When you press enter it will complete the selection with the first user in the list. This is by design.

**Comment by Datrogh** [comment]
> when you type a letter in the invite section it will provide the the list with the filter with the provided letter. You need to select the user from the filtered list or you need to enter the mail id. When you press enter it will complete the selection with the first user in the list. This is by design.

I seems I didn't communicate the problem correctly. I'll try again and I can provide a video of it as well (which contains colleague's email addresses, so I won't share it publicly)

Here the steps to reproduce the issue:

1. I open the calendar, create a new event and go to "add participants" ("Erforderliche Teilnehmer hinzufügen")
2. Now I enter the characters "sch" and view the list of options. It gives me 5 colleagues whose names contain the "sch". The Colleague "Schüssler" is not on the list, because he is a "Guest". (I know that it's not related to that because my name contains an "ü" as well and my colleagues have the same problem when trying to invite me)
3. As soon as I enter the next letter ("ü"), the first entry Teams found after entering the characters "sch" is selected as participant.
4. This does not happen with other "Umlaute", like "ä" or "ö", so it either has to do with "ü" being something like a shortcut for "übernehmen" ("select" in german) or the "ü" being on the same key as the US "[{"

You should try using the german keybord settings and entering "ü"

To be clear: There does not have to be a valid option with a "ü" in the name for this to problem to surface. "ü" seems to work exactly like "Return" in this screen...

**Comment by Datrogh** [comment]
Could you elaborate please what you mean with "any Alphabet default"? I don't understand the "by design" here. Especially because it wasn't that way when we started using MSTeams, this change came with some update.

**Comment by Nikitha-MSFT** [comment]
I have updated my comment. Please check

**Comment by Datrogh** [comment]
I'm sorry, but I still don't understand. What do you mean by "if you use any Alphabet"? Do you mean that this specific key on any keyboard works as a "Return"? If so, why? What is the purpose of a second "Return" key so close to the real "Return" key?

**Comment by Nikitha-MSFT** [comment]
Could you please try typing "a" and press enter and see the result, the first user from the search list id filled. so if you type any alphabet like "a"/"b"/ "ü" when you click on enter the first user from the list are filled in attendee box.

**Comment by Datrogh** [comment]
You still misunderstand the problem. Please reread what I described. When I use any keys, it just fills the seach window. As soon as I TRY to enter "ü" it just reacts AS IF I used "Enter", but the "ü" isn't considered for the search.
I want to add "ü" to the search and I DON'T press "Enter", but it still uses the letters BEFORE the "ü", which is wrong.

**Comment by Nikitha-MSFT** [comment]
Could you please share a screen recording of the issue for better understanding?

**Comment by Datrogh** [comment]
Yes, absolutely. I'd like to share it with you, but since there are email-addresses of my colleagues visible, I'd prefer to not share it here. How can I send you the link privately?

**Comment by Nikitha-MSFT** [comment]
Could you please drop a mail id :microsoftteamsdev@microsoft.com

**Comment by Datrogh** [comment]
done

**Comment by Nikitha-MSFT** [comment]
We haven't received any mail from you? could you please drop a mail again?

**Comment by Datrogh** [comment]
I sent it on April 6th 1:54 PM
![image](https://user-images.githubusercontent.com/37664442/115028978-c3767b80-9ec5-11eb-8c1b-dbb46d6a9bab.png)

I now resent it from a different mail account, maybe that helps

**Comment by Nikitha-MSFT** [comment]
Thank you received a mail, Tracking it on email.
